hack/build-local-images.py

```diff
@@ -207,6 +219,10 @@ def debug(message):
 
     config = image_config[image]
     for binary in config.get("binaries", []):
+        if "vendor_dir" in config:
+            os_bin_path = join(os_root,
```

Comment:
if this is now specific to the image config don't calculate it above, handle both cases here

Reply:
It's specific to any image config with a vendor_dir key. I agree that it'd be clearer to have the os_bin_path joins right next to each to other.